doc/latex.rst

```diff
@@ -111,7 +111,7 @@ If non-empty, it will be passed as argument to the ``\sphinxsetup`` command::
      ``\sphinxsetup``.
 
    - As an alternative to the ``'sphinxsetup'`` key, it is possibly
-     to insert explicitely the ``\\sphinxsetup{key=value,..}`` inside the
+     to insert explicitly the ``\\sphinxsetup{key=value,..}`` inside the
```

Comment:
on second thoughts I wonder if it would not be even better without any "explicitly" ?